docker/images/dev-cli/Dockerfile

```diff
@@ -200,6 +200,9 @@ RUN cd /tmp/; \
 ENV GOLANGCI_LINT 1.10.2
 RUN curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v$GOLANGCI_LINT
 
+# Install gocovmerge
+RUN go get -u github.com/wadey/gocovmerge
```

Comment:
Do not create a new FS layer for this ... should be executed with the other commands installing go tools